tests/modellauncher.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
+<pecan>
+  <outdir>pecan</outdir>
+
+  <database>
+    <bety>
+      <user>bety</user>
+      <password>bety</password>
+      <host>postgres</host>
+      <dbname>bety</dbname>
+      <driver>PostgreSQL</driver>
+      <write>FALSE</write>
+    </bety>
+    <dbfiles>/data/dbfiles/dbfiles</dbfiles>
+  </database>
+
+  <host>
+    <name>localhost</name>
+    <modellauncher>
+      <binary>/usr/local/bin/modellauncher</binary>
```

Comment:
@robkooper does this test depend on these options/settings being available on the local machine or is this just a VM test?

Reply:
The file binary needs to be located on the HPC cluster, not on the VM or on the local machine.